service-api/app/src/App/src/Service/Lpa/LpaService.php

```diff
@@ -119,11 +119,7 @@ public function getByUserLpaActorToken(string $token, string $userId): ?array
             $actor = ($this->resolveActor)($lpaData, $map['ActorId']);
 
             // If an active attorney is not found then we should not return an lpa
-            if ($actor !== null) {
-                $result['actor'] = $actor;
-            } else {
-                return null;
-            }
+            ($actor !== null) ? $result['actor'] = $actor : $result['actor'] = null;
```

Comment:
Surely this is the same as just doing
```suggestion
            $result['actor'] = $actor
```